### PR TITLE
OLS-139: Adds transcripts collection.

### PR DESCRIPTION
**Pull request by onmete**
## Description

Adds transcripts collection.

## Type of change

- [x] New feature

### CODE REVIEW

**Comment by bparees** [comment]
small tweaks, generally lgtm other than needing a rebase.

File: ols/app/endpoints/ols.py [hunk 1]
```diff
@@ -296,3 +308,49 @@ def generate_bare_response(conversation_id: str, llm_request: LLMRequest) -> str
...
+
+    Args:
+        user_id: The user ID (UUID).
+        conversation_id: The conversation ID (UUID).
+        validation_result: The result of the validation.
+        llm_request: The request containing a query.
+        response: The response to store.
+    """
+    # ensures storage path exists
+    storage_root = Path(config.ols_config.user_data_collection.transcripts_storage)
```
[the remark this answers is not in the document]
Comment (onmete) [reply ols/app/endpoints/ols.py:330]: yup

**Review by tisnik — COMMENTED** [review]
it looks good

**Pull request merged**